bin/create-tokens/main.go

```diff
@@ -25,8 +25,7 @@ const (
 	defaultValidWeeks = 24
 )
 
-var grantSignatorPrivateKeyHex = os.Getenv("GRANT_SIGNATOR_PRIVATE_KEY")
-
+var grantSigningKey = flag.String("grant-signing-key", "grant-signing-key", "a key to store the new public and private keys against")
```

Comment:
I would suggest tweaking the description, maybe something like `name of vault transit key to use for signing`